src/main/java/io/quarkus/gizmo2/MethodTyped.java

```diff
@@ -52,6 +53,13 @@ default boolean isPrimitiveReturn() {
         return returnType().isPrimitive();
     }
 
+    /**
+     * {@return the list of parameter types}
+     */
+    default List<ClassDesc> parameterList() {
```

Comment:
These methods probably should be called `parameterTypes()`; they are not a list of parameters but their types.

Reply:
I called it `parameterList()` because it delegates to a method called `parameterList()`, like all other methods in this interface. I can rename it, but preserving the name felt best.